packages/wdio-appium-service/src/launcher.ts

```diff
@@ -154,37 +154,70 @@ export default class AppiumLauncher implements Services.ServiceInstance {
             this._process.kill()
         }
     }
-
-    private _startAppium(command: string, args: Array<string>) {
+    private _startAppium(command: string, args: Array<string>, timeout:number = 60000) {
```

Comment:
```suggestion
    private _startAppium(command: string, args: Array<string>, timeout = 60000) {
```

Reply:
Can we move this into a `const` variable to the top of the file, e.g. `const APPIUM_START_TIMEOUT = 10 * 1000`

Should 10s be sufficient here?

Reply:
> Can we move this into a `const` variable to the top of the file, e.g. `const APPIUM_START_TIMEOUT = 10 * 1000`
> 
> Should 10s be sufficient here?

i suggest to add 30 sec. It's not a option that we can change now, so we need to be flexible. I agree with you with the const idea.